### ISSUE TITLE
Second Modification To Database Fails

### ISSUE THREAD

**Issue by TechD123** [Kunzisoft/KeePassDX#1444]
On Android 13 with a database saved via Nextcloud, only the first modification to the database is successfully saved to Nextcloud. Any modification afterwards results in an error (database could not be saved). Behavior is observed when creating, deleting or moving an entry, for example. Modern unlocking is enabled.

This issue appeared today. The only things I did differently, or that I can remember modifying are:
- Database security setting changed via KeepasXC Desktop Client

The first time this behavior was observed:
- Trying and failing to delete multiple entries (second modification) after having copied them to a different folder (first modification)

**Comment by J-Jamet** [comment]
This problem comes from Nextcloud which breaks the write permissions on the second opening of the file with the same URI. We should study why this application has this behavior.

**Comment by jlippuner** [comment]
I'm having the same issue. Does anybody know a workaround or fix? What can I do to help diagnose the issue?

**Comment by TechD123** [comment]
As the out of scope label was added and no progress has been made, should I bring this up with Nextcloud?

**Comment by J-Jamet** [comment]
Yes, the behavior you describe is the one that Nextcloud has set up for the file provider of its application.

**Comment by fuzunspm** [comment]
same issue

**Comment by DiagonalArg** [comment]
As I'm suffering with the same issue, I'm wondering if it is possible that giving us the option of setting a db-save interval that's a little longer than the repeated immediate saves that happen when we make a flurry of changes, might solve the issue?  Maybe if we could set it as once in 60sec or in 5min, it might not trigger the problem with nextcloud?
